src/client.rs

```diff
@@ -118,6 +120,18 @@ mod tests {
         crate::tests::setup_tracing();
 
         let tempdir = TempDir::new("tor-cache").expect("create temp dir");
+
+        let consensus = include_str!("test-data/consensus.txt");
+        let microdescriptors = include_str!("test-data/microdescriptors.txt");
+
+        let consensus_path = tempdir.path().join("consensus.txt");
+        let mut consensus_file = File::create(consensus_path).expect("create temp consensus file");
+        write!(consensus_file, "{}", consensus).expect("write temp consensus file");
+
+        let microdescriptors_path = tempdir.path().join("microdescriptors.txt");
+        let mut microdescriptors_file = File::create(microdescriptors_path).expect("create temp microdescriptors file");
+        write!(microdescriptors_file, "{}", microdescriptors).expect("write temp microdescriptors file");
```

Comment:
As it is already implemented in `src/arti.rs`, consider moving it to `src/tests.rs`.